promtimer/promtimer.py

```diff
@@ -252,7 +285,63 @@ def main():
     grafana_port = args.grafana_port
     prometheus_base_port = grafana_port + 1
 
-    if not args.cluster:
+    cbbackupmgr_stats_mode = False
+    buckets = None
+
+    if args.stats_archive_path:
+        cbbackupmgr_stats_mode = True
+
+        archive_path = args.stats_archive_path
+        if zipfile.is_zipfile(args.stats_archive_path):
```

Comment:
perfectly fine to stay with zipfile if that's consistent with other parts of promtimer, but it may be worth bearing in mind that the zipfile module is pure python, and _rather_ slow. If you're expecting large archives, it may be good to shell out to unzip (and fall back to the python module if necessary).

Buttt, that might be out of scope for this change 👍

Reply:
As discussed, that's currently out of scope but I will keep it in mind for any future work :)